test/unittests/main.cpp

```diff
@@ -0,0 +1,179 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2019 The evmone Authors.
+// Licensed under the Apache License, Version 2.0.
+
+#include "vm_loader.hpp"
+#include <evmc/loader.h>
+#include <gtest/gtest.h>
+#include <iostream>
+#include <string>
+#include <vector>
+
+/// The loaded EVMC module.
+static evmc::vm evmc_module;
```

Comment:
It would be less confusing if this and `get_evm` below were in a separate cpp file, then it would be more clear that it's another implementation for `get_evm` of `vm_loader.hpp`
(some setter I guess would be required then, too)

Reply:
It was planned like that, but it gets tricky because `evmc_module` is set from `main()`.